tests/test_session_camera_api.py

```
@@ -0,0 +1,288 @@
+            f"{server}/api/rooms/{room_id}/geometries/{camera_key}",
+            headers=conn.headers,
+            timeout=10,
+        )
+        if response.status_code == 404:
+            break
+        time.sleep(poll_interval)
+        elapsed += poll_interval
+    else:
+        raise AssertionError(
+            f"Session camera not deleted within {max_wait}s after disconnect"
+        )
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)